### ISSUE TITLE
Does not read zip files

### ISSUE THREAD

**Issue by sadikovi** [sadikovi/parsebox#1]
Hadoop InputFormat fails reading zip files.

**Comment by sadikovi** [comment]
Consider adding `ZipCodec` to the Hadoop ompression codec factory.

**Comment by sadikovi** [comment]
``` scala
val df = pc.format("com.github.sadikovi.parsebox.examples.dns").
    load("file:/test/sample.zip")
```

**Comment by sadikovi** [comment]
Closed in 57df8a9787613542e7594e43147f697c00e94e99
